### PR TITLE
Add missing translations for Afrikaans

### LINKED ISSUE #38720: Missing translations for Afrikaans (af)
Hello,

There are some missing translation for Afrikaans (af). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.af.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.af.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.af.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.af.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.af.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.af.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by pierredup**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38720
| License       | MIT
| Doc PR        | N/A

Add missing translations for Afrikaans.

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.af.xlf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                <source>Please enter an integer.</source>
+                <target>Voer asseblief 'n geldige heeltal in.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Kies assblief 'n geldige taal.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Voer assebliefn 'n geldige landstreek in.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.af.xlf]: Does this refer to a locale like "en", "af", "fr" etc?

**Review by Nyholm — COMMENTED** [review]
Awesome. Thank you. 

Could you just double check the one comment? I used google translate, Im probably wrong but I would like you to confirm.

File: src/Symfony/Component/Form/Resources/translations/validators.af.xlf (hunk 1 above)
Comment (pierredup) [reply src/Symfony/Component/Form/Resources/translations/validators.af.xlf]: There isn't really anything with the same meaning in Afrikaans, this refers more to something like a `Language area`, which might not be the same thing. I see some other translations just left it as `locale`, so I made the same change here.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.af.xlf]: Okey. 
As long as it makes sense. 

Thank you 👍

**Review by Nyholm — APPROVED** [review]
I updated the base and milestone to 4.4. It is not ready for a final review and merge. 

Thank you @pierredup for you work!

**Comment by derrabus** [comment]
Thank you @pierredup.

**Pull request merged**